server/Program.cs

```diff
@@ -67,63 +63,40 @@
     options.DefaultPolicy = defaultPolicy;
     options.FallbackPolicy = defaultPolicy;
 });
-builder.Services.AddControllersWithViews();
 
-builder.Services.AddCors(options =>
-{
+// TODO
```

Comment:
If this is for having a local database, do a check on whether or not the environment is development, and set DbContext based on this

Reply:
added environment check